common/cpp/verify_ias_report/CMakeLists.txt

```diff
@@ -42,19 +42,22 @@ add_custom_command(OUTPUT  ${PROJECT_GENERATED_IAS_SOURCES}
 # Untrusted Verify-Ias-Report Library
 ################################################################################
 
+SET(SGX_SDK "$ENV{SGX_SDK}")
+SET(SGX_SDK_INCLUDE ${SGX_SDK}/include)
```

Comment:
Do we need SGX SDK dependency for untrusted part.? SGX SDK dependency previously was based on the condition `if(NOT UNTRUSTED_ONLY)`.